dev/ManagedConnectors.md

```diff
@@ -144,6 +144,89 @@ ocm login --token <offline_token>
 ./cos-tools/bin/create-cluster-secret <cluster_id>
 ```
 
+### Step 6: Update Job
+
+There's no automatic CI/CD mechanism in Managed Connectors to update the operators with the newest version.
+
+The only way at the moment is to configure a Kubernetes [CronJob](https://v1-21.docs.kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/)
+that kills the existing operator pods, so that the deployment triggers the pull of the newest image (if existing) before recreating it.
+
+To configure this job, create a file named `updatejob.yaml` with the following content:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cos-updater
+  namespace: cos
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pod-killer
+  namespace: cos
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - pods
+    verbs:
+      - get
+      - watch
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cos-updater-pod-killer
+  namespace: cos
+subjects:
+  - kind: ServiceAccount
+    name: cos-updater
+    namespace: cos
+roleRef:
+  kind: Role
+  name: pod-killer
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cos-update-by-delete
+  namespace: cos
+spec:
+  concurrencyPolicy: Forbid
+  schedule: "@hourly"
```

Comment:
@kostola What about we have this run once per day, Mon-Fri, before everyone gets into the office, so perhaps 7 a.m UTC. That means we can start the day with the latest version of MC and then verify it's still running before we start the working day.

WDYT?

Reply:
Good idea...I will update this :+1:

Reply:
updated :+1: